### ISSUE TITLE
mistaken release 4.6.1?

### ISSUE THREAD

**Issue by crotwell** [tildeio/rsvp.js#496]
Looks like 4.0.1 is the latest, but there is a 4.6.1 in the github releases list. Not big deal but kind of confusing.

Also,  a search for rsvp on npmjs.com shows 4.0.1 to be the latest, but 4.6.1 shows up if I do
```
npm outdated
Package                  Current  Wanted  Latest  Location
rsvp                       4.0.1   4.6.1   4.0.1  xxxxxxxx
```

I am not sure what you can do, but would be nice if 4.6.1 could be unpublished, or at least deprecated, from the npm registry.

**Comment by crotwell** [comment]
`npm view rsvp versions` shows
```
...
  '3.5.0',
  '3.6.0',
  '3.6.1',
  '3.6.2',
  '4.0.0',
  '4.0.1',
  '4.6.1' ]
```

**Comment by Turbo87** [comment]
@tilde-engineering @stefanpenner @rwjblue  this is trolling quite a few people (see https://github.com/ember-cli/ember-cli/pull/7345 for example). please make sure to remove that version from npm or if not possible release a `4.6.2` that equals `4.0.2` (or give me push access then I'll do it...)

**Comment by rwjblue** [comment]
Published 4.0.2 as 4.7.0.
